libcob/common.h

```diff
@@ -1395,7 +1395,7 @@ typedef struct __cob_file {
 	cob_file_key		*keys;			/* ISAM/RANDOM/SORT keys */
 	void			*file;			/* File specific pointer */
 	void			*linorkeyptr;		/* LINAGE or SPLIT KEY */
-	const unsigned char	*sort_collating;	/* SORT collating / CODE-SET (used for RE-/WRITE) */
+	const unsigned char	*sort_collating;	/* SORT collating */
```

Comment:
please adjust comment as this is now also the FILE collation for INDEXED files

Reply:
Actually canceled this change, since the file collation is now directly stored in the keys.